modules/aleph/src/Aleph/Handler/AlephPatronHandler.php

```diff
@@ -255,6 +262,22 @@ public function deleteReservation($patron, $reservation) {
     return AlephRequestResponse::createRequestResponseFromXML($response);
   }
 
+  /**
+   * @param AlephPatron $patron
+   * @param AlephMaterial $material
+   *
+   * @return AlephHoldGroup[]
+   * @throws \RuntimeException
```

Comment:
Throwing runtime-exceptions is a bit like saying "I might have messed up, and now it's your problem". Use a business-exception instead (eg. a Aleph exception) or handle the issue and return something well-defined such ash NULL, FALSE or []